src/pages/configure/configure.jsx

```diff
@@ -54,6 +58,30 @@ const Configure = () => {
 			.finally(() => dispatch(decrement("pendingRequests")))
 	}
 
+	const removeProjectConfig = () => {
+		return new Promise((resolve, reject) => {
+			store.dispatch(increment("pendingRequests"))
+			client.projects.deleteProject(projectID).then(() => {
+				notify("success", "Success", "Removed project config successfully")
+				const dbConfig = store.getState().extraConfig
+				const dbList = delete dbConfig[projectID]
+				store.dispatch(set(`extraConfig`, dbList))
+				const projectConfig = store.getState().projects;
+				const projectList = projectConfig.filter(project => {
+					if(project.id !== projectID) return project
```

Comment:
The function passed to `.filter()` should always return `true/false`. In cases where the filtering logic is simple as checking a condition you can do it like these:

```
.filter(project => project.id === projectId)
```